src/entity/Course.java

```diff
@@ -0,0 +1,50 @@
+package entity;
+
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Course extends Activity {
```

Comment:
codul nu compileaza, Activity pare sa nu mai existe?

Reply:
Am adaugat acum manual clasele care lipseau din etapa2 fata de etapa1. Nu stiu de ce s-a intamplat sa nu mai fie.. La mine compileaza local. Daca inca nu este in regula cu pull request si github, pot trimite arhiva.. Imi cer scuze, tot am probleme cu folosirea github-ului :(

Reply:
acum pare sa fie ok, sa compileze :) baby steps, daca iti propui, inveti si git/github mai bine cu timpul :) scopul e sa va obisnuiti de pe acum